ibis/backends/dask/__init__.py

```diff
@@ -10,3 +20,63 @@
         'See ibis.dask.trace for details.',
         validator=ibis.config.is_bool,
     )
+
+
+def connect(dictionary):
+    """Construct a dask client from a dictionary of DataFrames.
+
+    Parameters
+    ----------
+    dictionary : dict
+
+    Returns
+    -------
+    DaskClient
+    """
+    return DaskClient(dictionary)
+
+
+def from_dataframe(df, name='df', client=None):
```

Comment:
I would suggest removing the default value "df" for `name` and make it required. 

If "client" is None, it seems like you created a Client object inside, but how does the user get an handler to that client?

Reply:
I agree in principle with your suggestions, but I think it makes sense to keep the top level interfaces of dask and pandas backends the same. I can create a new issue to fix this in both backends together?

I think you can grab the client off of the `source` variable on the table objects, but agreed that's not user friendly.